assignment-1/cause_2.md

```diff
@@ -0,0 +1,68 @@
+# Architecture - rigid and lacking non-functional requirements
+
+## Key factors in failure
+
+One of the primary problems with HealthCare.gov was its performance.
+The architecture was designed to pass on requests from the web-servers real-time to the back-end systems.
```

Comment:
Based on [dictionary.com](http://dictionary.reference.com/browse/web-server?s=t), web server does not have any dash